assets/src/components/minimalLadder.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useContext } from "react"
+import RouteLadders from "./routeLadders"
+import { routeTabById } from "../models/routeTab"
+import { StateDispatchContext } from "../contexts/stateDispatchContext"
+import {
+  deselectRouteInTab,
+  flipLadderInTab,
+  toggleLadderCrowdingInTab,
+} from "../state"
+import { useParams } from "react-router-dom"
+
+export const MinimalLadder = () => {
+  const [{ routeTabs }, dispatch] = useContext(StateDispatchContext)
+  const id = useParams().id as string
+
+  const routeTab = routeTabById(routeTabs, id)
+  if (!routeTab) window.location.href = "/minimal"
```

Comment:
suggestion: can we use [`useNavigate`](https://reactrouter.com/en/main/hooks/use-navigate)?